backend/Squirrel.Core/Squirrel.Core.Common/DTO/Users/UserDTO.cs

```diff
@@ -0,0 +1,10 @@
+﻿namespace Squirrel.Core.Common.DTO.Users
+{
+    public class UserDTO
+    {
+        public int Id { get; set; }
+        public string Avatar { get; set; }
```

Comment:
do we really need avatar right now? this info is not passed for auth

Reply:
we will need avatar later